common/e2e_config/e2e_config.go

```diff
@@ -137,6 +137,7 @@ type ProductSpec struct {
 	LvmEnginePluginContainerName      string            `yaml:"lvmEnginePluginContainerName"`
 	LvmEnginePluginDriverName         string            `yaml:"lvmEnginePluginDriverName"`
 	UmbrellaOpenebsHelmChartName      string            `yaml:"umbrellaOpenebsHelmChartName"`
+	UseUmbrellaOpenEBSChart           bool              `yaml:"useUmbrellaOpenEBSChart" env-default:"false"`
```

Comment:
please add the name of the environment variable which can be set to control this setting.
For an example see `PlatformConfigFile` in E2EConfig

Reply:
Added